content/departments/product-engineering/engineering/cloud/devops/resize-disks-in-statefulset.md

```diff
@@ -0,0 +1,69 @@
+# Resize disks in GKE StatefulSet
+
+Important: this change will caused recreation on all StatefulSet's pods, which can cause donwtime if application is not high-available.
```

Comment:
Typo fixes and rephrased to indicate that this is specific to the StatefulSet being modified.

```suggestion
Important: this change will cause recreation of all pods in the StatefulSet, which can cause downtime if application is not highly available.
```